.github/stale.yml

```diff
@@ -4,13 +4,12 @@ daysUntilStale: 90
 daysUntilClose: 14
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - regression
-  - future
-  - feature
-  - enhancement
-  - confirmed
+  - "Type: Regression"
+  - "Type: Discussion"
+  - "Type: Enhancement"
+  - "Status: Confirmed"
```

Comment:
I might have missed some cases here.

Here is the current tentative list of labels (subject to some changes as we use them over the next few weeks):

- Type: Bug
- Type: Maintenance
- Type: Enhancement
- Type: Question
- Type: Discussion
- Type: Regression
- Blocked: Backend
- Blocked: Frontend
- Status: Completed
- Status: Merge Conflict
- Status: In Progress
- Status: On Hold
- Status: Review Needed
- Status: Abandonned
- Status: Stale
- Status: Confirmed
- Status: Needs Triage
- Priority: Critical
- Priority: High
- Priority: Medium
- Priority: Low
- Scope: Vue Migration
- Scope: ES6 Migration
- Scope: Cleanup
- Scope: UI Redesign
- Scope: New Feature
- Scope: Bug
- Scope: Build & CI
- Scope: Dependencies
- Scope: Security
- Misc: Help Wanted
- Misc: Stable Backport
- Misc: Discussion Needed
- Misc: Duplicate

If you spot one that needs to be spared by stalebot but I haven't included, I'm all ears.